### ISSUE TITLE
Xcode 13.3: Umbrella header for module 'MappIntelligenceSDK_iOS' does not include header 'MIFormField.h'

### ISSUE THREAD

**Issue by juanjovalino4242** [mapp-digital/MappIntelligence-iOS-v5#1]
Got an error when updating to Xcode 13.3 and Swift 5.6 at `MappIntelligenceSDK_iOS.h`:

`Umbrella header for module 'MappIntelligenceSDK_iOS' does not include header 'MIFormField.h'`

**Comment by MappDeveloper** [comment]
Hi, 
We found that error for Carthage installation and we will fix it with new version, until then feel free to use cocoa pods or SPM ( with those it works perfectly).

**Comment by victorrodriguez2016** [comment]
Hi,

In our case it is not working for swift package manager either. Is there any special configuration needed for it to work?

This is blocking us for a while now, do you have a release date?

On the meantime, do you have a branch where we could point to?

Thanks

**Comment by victorrodriguez2016** [comment]
Morning,

do you have an update on this?

Best,
myToys iOS Team

**Comment by StefanMiticon** [comment]
Morning, 
@victorrodriguez2016, @juanjovalino4242 

Can you try branch 'carthage_test' to see if it works for you? 
After that we will include that changes in our next official release. 

Best Regards,
Stefan, Mapp Digital team

**Comment by victorrodriguez2016** [comment]
Morning Stefan,

I tried and I get this error:

```
➜  MappTest git:(main) ✗ carthage update --use-xcframeworks
*** Fetching MappIntelligence-iOS-v5
*** Checking out MappIntelligence-iOS-v5 at "50cf8248cffa259b18186c4d2cd60a782ab3f326"
*** xcodebuild output can be found in /var/folders/_z/s47mt_8s7nd67qj9hwqqpxj9bz4hyl/T/carthage-xcodebuild.pN2yhy.log
*** Building scheme "MappIntelligenceSDK-iOS" in MappIntelligenceSDK.xcworkspace
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -workspace /Users/vrodriguez/Desktop/MappTest/Carthage/Checkouts/MappIntelligence-iOS-v5/MappIntelligenceSDK.xcworkspace -scheme MappIntelligenceSDK-iOS -configuration Release -derivedDataPath /Users/vrodriguez/Library/Caches/org.carthage.CarthageKit/DerivedData/13.4_13F17a/MappIntelligence-iOS-v5/50cf8248cffa259b18186c4d2cd60a782ab3f326 -sdk iphonesimulator -destination platform=iOS\ Simulator,id=7B340F98-239A-4599-B728-4899E7176564 -destination-timeout 3 ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES build VALIDATE_WORKSPACE=NO (launched in /Users/vrodriguez/Desktop/MappTest/Carthage/Checkouts/MappIntelligence-iOS-v5)

This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/_z/s47mt_8s7nd67qj9hwqqpxj9bz4hyl/T/carthage-xcodebuild.pN2yhy.log
```

Thanks for looking into it,
Victor

**Comment by victorrodriguez2016** [comment]
I created this fork https://github.com/victorrodriguez2016/MappIntelligence-iOS-v5-fix where I was able to somehow make it work, but I'm not sure if this changes are correct or not.
Just in case it helps :)

**Comment by victorrodriguez2016** [comment]
Hi,

I would like to as what's the status of the changes requested for the cartage_test branch, as we sent a few emails but haven't got a response yet.

Thanks,
Victor

**Comment by StefanMiticon** [comment]
Hi Victor, 

we have created new branch for test carthage_new, please try it. If that still does not working I would like to have one more debugging session. 

Best Regards,
Stefan Stevanović

**Comment by victorrodriguez2016** [comment]
Hi Stefan,

we are still facing the same issue. Let's schedule a debugging session.

<img width="1774" alt="Screenshot 2022-06-27 at 15 43 38" src="https://user-images.githubusercontent.com/62607492/175956156-0f9d612c-90a6-4b34-974d-5b02f36a3ac2.png">

Best Regards,
Victor

**Comment by StefanMiticon** [comment]
The problem was resolved with 5.0.3.2 version. 
Thanks a lot to @victorrodriguez2016 which helps me to resolve this issue. 
I am happy to close this issue.

**Comment by victorrodriguez2016** [comment]
Thanks for all the effort @StefanMiticon!
